datadog_checks_dev/datadog_checks/dev/tooling/manifest_validator/v2/validator.py

```diff
@@ -34,6 +34,19 @@ def validate(self, check_name, decoded, fix):
                 self.fail(output)
 
 
+class TileDescriptionValidator(BaseManifestValidator):
+    DESCRIPTION_PATH = '/tile/description'
+    MAX_DESCRIPTION_LENGTH = 70
+
+    def validate(self, check_name, decoded, fix):
+        # The description for V2 manifests should not be longer than 70 characters to avoid being
+        # cut off or shortened on the UI
+        tile_description = decoded.get_path(self.DESCRIPTION_PATH)
+        if len(tile_description) > self.MAX_DESCRIPTION_LENGTH:
+            output = f'  The tile description should be no longer than {self.MAX_DESCRIPTION_LENGTH} characters.'
```

Comment:
Super tiny nit, can we log the current value in this error? 
Suggestion that I think would make this feel more actionable:

"The tile description is currently {} characters long. This must be no longer than {} characters"